Szwalnia/SzacowanyCzas.cs

```diff
@@ -45,20 +45,25 @@ private void btnObliczDate_Click(object sender, EventArgs e)
             czasTeraz = DateTime.Now;
             czasTeraz2 = czasTeraz.TimeOfDay;
             godziny = czasTeraz2.Hours;
+            int ostatniaGodzinaPracy = 22;
+            int czasPracy = 16;
+            int dobowaPrzerwa = 8;
+            int weekend = 48;
+            int piatek = 5;
 
-            if ((22 - godziny) > czasWieluSztukInt)
+            if ((ostatniaGodzinaPracy - godziny) > czasWieluSztukInt)
             {
                 czasObliczeniowy = czasWieluSztukInt;
             }
             else {
-                czasObliczeniowyDni = (czasWieluSztukInt - (22 - godziny)) / 16 + 1; 
+                czasObliczeniowyDni = (czasWieluSztukInt - (ostatniaGodzinaPracy - godziny)) / czasPracy + 1; 
```

Comment:
o wiele lepiej się teraz to czyta, prawda? :)

Reply:
zgadza się

Reply:
Rozwiązano konflikty